sql-cli/tests/test___main__.py

```diff
@@ -165,6 +197,32 @@ def test_validate_all(initialised_project_with_test_config):
     assert output.startswith("Validating connection(s)")
 
 
+@pytest.mark.parametrize(
+    "env,connection,status,message",
+    [
+        ("test", "sqlite_conn_invalid", "FAILED", "Sqlite db does not exist!"),
+    ],
+)
+def test_validate_verbose(env, connection, status, message, initialised_project_with_test_config):
+    result = runner.invoke(
+        app,
+        [
+            "validate",
+            initialised_project_with_test_config.directory.as_posix(),
+            "--env",
+            env,
+            "--connection",
+            connection,
+            "--verbose",
+        ],
+    )
+    assert result.exit_code == 0
+    output = get_stdout(result)
+    assert f"Validating connection(s) for environment '{env}'" in output
+    assert f"Validating connection {connection:{CONNECTION_ID_OUTPUT_STRING_WIDTH}} {status}" in output
+    assert f"  {message}" in output
```

Comment:
Are there any other statements we'd expect to see when verbose is enabled for the `validate` command?

Reply:
I don't think so right now. If you want to see airflow logs, you would additionally need to run with `--debug`.